.gitignore

```diff
@@ -140,3 +140,7 @@ Pipfile.lock
 
 # No Pycharm
 .idea/
+.DS_store
```

Comment:
this is harmless but did you mean to include this?

It's best to treat each PR as a separate concern - if you want to modify the .gitignore do it as a separate PR (but this is OK for now)

Reply:
@victoriasoesanto - please don't delete the .gitignore file from this PR.  deleting the Pipfile will be good, but not the .gitignore.